reconcile/integrations_manager.py

```diff
@@ -0,0 +1,131 @@
+import os
+import sys
+import json
+
+from typing import Any, Dict, List, Mapping
+
+import reconcile.openshift_base as ob
+
+from reconcile.utils import helm
+from reconcile import queries
+from reconcile.status import ExitCodes
+from reconcile.utils.oc import OCDeprecated, OC_Map
+from reconcile.utils.semver_helper import make_semver
+from reconcile.utils.openshift_resource import OpenshiftResource, ResourceInventory
+from reconcile.utils.defer import defer
+
+
+QONTRACT_INTEGRATION = "integrations-manager"
+QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
+
+
+def construct_values_file(
+    integration_specs: List[Mapping[str, Any]]
+) -> Mapping[str, Any]:
+    values: Dict[str, Any] = {
+        "excludeService": True,
```

Comment:
i'm a bit puzzled about this flag. so the service for metrics scraping is only rolled out as part of the manager deployment and this flag prevents rolling it out a second time as part of the managed integrations?

what about the situation where the manager is rolled out to a different namespace than the integrations?